backend/experiment/rules/rhythm_battery_intro.py

```diff
@@ -17,7 +17,22 @@ def next_round(self, session):
         round_number = session.get_rounds_passed()
         playback = None
         feedback_form = None
+        actions = []
         if round_number == 0:
+            explainer = Explainer(
+                instruction=_(
+                    'General listening instructions:'),
+                steps=[
+                    Step(_(
+                        "To make sure that you can do the experiment as well as possible, please do it a quiet room with a stable internet connection."),
+                    ),
+                    Step(_("Please use headphones, and turn off sound notifications from other devices and applications (e.g., e-mail, phone messages)."),
+                         )],
+                step_numbers=True,
+                button_label=_('Ok')
+            )
+            actions.append(self.intro_explainer())
+            actions.append(explainer)
```

Comment:
All other methods just return the one explainer object. I'm wondering whether it would be good to add and document this method to `base.py`, but then we would *always* have an Explainer in place unless we override.